README.md

```diff
@@ -83,7 +83,7 @@ cargo run --bin indexer -- \
 load-osm-pbf /data/australia-oceania-latest.osm.pbf
 
 # Run service
-cargo run --bin airmail_service -- \
```

Comment:
I'm surprised this works for you?  Is it worth making that the default bin or renaming accordingly?

```bash 
cargo run --bin airmail -- \
--index /data/index/
error: no bin target named `airmail`.
Available bin targets:
    indexer
    merge
    query
    airmail_service
```

Reply:
Ooooooops, I misread the instructions apparently.